Lib/test/test_pathlib.py

```diff
@@ -1679,6 +1682,9 @@ def _check(glob, expected):
         p = P(BASE, "dirC")
         _check(p.rglob("file*"), ["dirC/fileC", "dirC/dirD/fileD"])
         _check(p.rglob("*/*"), ["dirC/dirD/fileD"])
+        # gh-91616, a re module regression
+        _check(p.rglob("*.txt"), ["dirC/novel.txt"])
+        _check(p.rglob("*.*"), ["dirC/novel.txt"])
```

Comment:
Would not testing `"*eC"` be enough?

Reply:
I would keep this.
`*.txt` is a common usage, but it's not tested.